packages/flutter/test/widgets/listener_test.dart

```diff
@@ -125,6 +129,7 @@ void main() {
       ));
       expect(exit, isNotNull);
       expect(exit.position, equals(const Offset(400.0, 300.0)));
+      expect(WidgetsBinding.instance.mouseTracker.isAnnotationAttached(renderListener.hoverAnnotation), isFalse);
```

Comment:
Instead, you can use `tester.binding.mouseTracker`...
```suggestion
      expect(tester.binding.mouseTracker.isAnnotationAttached(renderListener.hoverAnnotation), isFalse);
```